packages/opencode/src/config/config.ts

```diff
@@ -438,6 +438,12 @@ export namespace Config {
 
   export const TUI = z.object({
     scroll_speed: z.number().min(1).optional().default(2).describe("TUI scroll speed"),
```

Comment:
Now that this is considered, I would default it to 1, because the default linear scroll speed in opentui is what some users expect now.

Reply:
done